apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx

```diff
@@ -92,8 +91,9 @@ export default function SendSelectRecipient({ navigation, route }: Props) {
           recipient,
         }),
       );
+      setValue(recipient);
     },
-    [account, parentAccount, setTransaction, transaction],
+    [account, parentAccount, setTransaction, transaction, setValue],
```

Comment:
`setValue` as a dependency ? Why not `value` ?

Reply:
it's a `useCallback` so value is the argument of that method, no need to have it as dep